nix/tools/withTools.nix

```diff
@@ -81,7 +81,8 @@ let
           # On MacOS, it's 104 chars
           # See: https://serverfault.com/questions/641347/check-if-a-path-exceeds-maximum-for-unix-domain-socket
 
-          pg_ctl -l "$tmpdir/db.log" -w start -o "-F -c listen_addresses=\"\" -c hba_file=$HBA_FILE -k $PGHOST -c log_statement=\"all\" " \
+          # max_locks_per_transaction was modified from the default so we can create a lot of tables in one transaction
+          pg_ctl -l "$tmpdir/db.log" -w start -o "-F -c listen_addresses=\"\" -c max_locks_per_transaction=512 -c hba_file=$HBA_FILE -k $PGHOST -c log_statement=\"all\" " \
```

Comment:
No coverage failure on https://github.com/PostgREST/postgrest/pull/4578, looks like the `max_locks_per_transaction` is the culprit here (although the logs don't say anything). It's not really needed so I'll remove.